.github/workflows/ci.yaml

```
@@ -33,9 +33,9 @@ jobs:
       fail-fast: false
       matrix:
         juju:
-          - agent: 2.9.46
-            libjuju: ^2
-          - agent: 3.1.7
+          - agent: 2.9.46  # renovate: latest juju 2
+            libjuju: ==2.9.46.1  # renovate: latest libjuju 2
+          - agent: 3.1.7  # renovate: latest juju 2
```

Shouldn't it be the following?
```suggestion
          - agent: 3.1.7  # renovate: latest juju 3
```